syntax = "proto3";

package checker;

service Checker {
  // Tests student code while keeping source opaque.
  rpc run_tests(TestRequest) returns (TestReply) {}
}

// The request message containing the user's name.
message TestRequest {
  string source = 1;
}

// The response message containing the greetings
message TestReply {
  string response = 1;
  optional int32 status = 2;
}
